theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/KeyframeDot.tsx

```diff
@@ -136,17 +152,54 @@ function useKeyframeContextMenu(
   })
 }
 
-function useDragKeyframe(
+/** The editor that pops up when directly clicking a Keyframe. */
+function useKeyframeInlineEditorPopover(props: IKeyframeDotProps) {
+  const editingTools = useEditingToolsForKeyframeEditorPopover(props)
```

Comment:
Might be a premature optimization, but I think we shouldn't run `useEditingTools...` unless the popover has been triggered.